packages/arb-token-bridge-ui/src/components/common/TestnetToggle.tsx

```diff
@@ -10,20 +10,26 @@ import { useNetworks } from '../../hooks/useNetworks'
 export const TestnetToggle = ({
   className,
   label,
-  description
+  description,
+  addModeText
```

Comment:
I wasn't sure at first what it refers to, could we improve the variable name a little bit?

Some suggestions that are clearer to me:
```
showToggleStatus
showToggleState
includeToggleStatus
includeToggleState
```

Feel free to modify it further, but the current one just doesn't click instantly.